Dockerfiles/Dockerfile.ssas

```diff
@@ -11,11 +11,13 @@ RUN go get -u github.com/derekparker/delve/cmd/dlv
 RUN go get github.com/BurntSushi/toml
 RUN go get github.com/howeyc/fsnotify
 RUN go get github.com/mattn/go-colorable
+RUN go get github.com/CMSgov/bcda-ssas-app/ssas/service/main
 
-WORKDIR /go/src/github.com/CMSgov/bcda-app/ssas
-COPY . .
+COPY --from=migrate/migrate /migrate /usr/local/bin/migrate
```

Comment:
Can we get away with the same technique we use in the `bcda-ssas-app` repo?  (i.e., a docker-compose file specifically for bringing `migrate/migrate` into the stack).   Then we don't have to unrealistically bake `migrate/migrate` into the SSAS service image.

Reply:
We can, but then we have to get the `.sql` migration files into that image.  Perhaps you would have better success than I did with that strategy.

What I like about this line is that it only copies one file.

Reply:
Do the `.sql` files exist in this image?  If so, we can just mount a volume when we spin up the container so that the dir with the `.sql` files in the container is shared with the host.

Reply:
Latest commit uses the same technique as the `bcda-ssas-app` repo